### ISSUE TITLE
[BUG] medium.en-int8 invalid choice for model

### ISSUE THREAD

**Issue by jaburges** [linuxserver/docker-faster-whisper#8]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

seems to be an additional variable

### Steps To Reproduce

select medium.en.int8

### Environment

```markdown
- OS: Unraid
```

### CPU architecture

x86-64

### Docker creation

```bash
via Unraid
```

### Container logs

```bash
__main__.py: error: argument --model: invalid choice: 'medium.en-int8' (choose from 'tiny', 'tiny-int8', 'base', 'base-int8', 'small', 'small-int8', 'medium', 'medium-int8')
usage: __main__.py [-h] --model
                   {tiny,tiny-int8,base,base-int8,small,small-int8,medium,medium-int8}
                   --uri URI --data-dir DATA_DIR [--download-dir DOWNLOAD_DIR]
                   [--device DEVICE] [--language LANGUAGE]
                   [--compute-type COMPUTE_TYPE] [--beam-size BEAM_SIZE]
                   [--debug]
__main__.py: error: argument --model: invalid choice: 'medium.en-int8' (choose from 'tiny', 'tiny-int8', 'base', 'base-int8', 'small', 'small-int8', 'medium', 'medium-int8')
```

**Comment by jaburges** [comment]
https://github.com/linuxserver/docker-faster-whisper/pull/9

**Comment by homerr** [comment]
Closed by #6
